### DISCUSSION TITLE
Redux Toolkit Integration

### DISCUSSION THREAD

**Discussion by indranildeveloper** [keplergl/kepler.gl#2805] [General]
As of now, if we use Redux Toolkit with Kepler.GL reducers while uploading files with data we get the `maximum call stack exceed` error in the browser. Most probably there might be some other reducers that are not yet compatible with Redux Toolkit.

Want to discuss how we can integrate Kepler.GL reducers with Redux Toolkit.

**Comment by ibgreen** [comment]
@indranildeveloper thanks for engaging in kepler.gl

@ilyabo might have some thoughts on this.

**Reply by ilyabo** [reply]
Hi Indranil! Not sure what could be causing this. It'd help if you could share a minimal example to reproduce the issue.
